app/controllers/miq_ae_class_controller.rb

```diff
@@ -1499,8 +1499,12 @@ def create_ns
       @in_a_form = false
       replace_right_cell
     when "add"
-      parent_id = @edit[:typ] == "MiqAeDomain" ? nil : from_cid(x_node.split('-')[1])
-      add_ae_ns = @edit[:typ].constantize.new(:parent_id => parent_id)
+      parent_id, tenant = if @edit[:typ] == "MiqAeDomain"
+                            [nil, current_tenant]
+                          else
+                            [from_cid(x_node.split('-')[1]), nil]
+                          end
+      add_ae_ns = @edit[:typ].constantize.new(:parent_id => parent_id, :tenant => tenant)
```

Comment:
do we have access to the parent namespace? can we also do parent_namespace.ae_domains.new() [ or is that unneeded looups]